pkg/virt-handler/vm.go

```diff
@@ -295,12 +295,22 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 			for _, domainInterface := range domain.Spec.Devices.Interfaces {
 				interfaceMAC := domainInterface.MAC.MAC
 				var newInterface v1.VirtualMachineInstanceNetworkInterface
+				var isMasqueredeInterface bool = false
 
+				for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
```

Comment:
How about iterating once through the interfaces and creating a map with the name? I would prefer if we avoid nested loops. What do you think? I know that most likely these arrays are not that big, but let's try to avoid n^2  runtime.

Reply:
Good suggestion, changed it now